### ISSUE TITLE
Define how we measure "success" of our website

### ISSUE THREAD

**Issue by james-nash** [jpmorganchase/salt-ds#533]
Putting a bunch of pages online is one thing, but how do we _know_ they are meeting their goals? What, exactly, are those goals anyway?

This task is for us to answer those questions and setup whatever we need in order to get the required feedback and/or data.

### Additional considerations
While the V1 website will be modest in terms of scope, it will nonetheless be _the_ place that all Odyssey users go for docs, help, inspiration, etc. We should therefore think about what kind of insights we could and would like to get. Those can then help inform future iterations of the website (and maybe even Odyssey as a whole).

**Note:** When thinking about what data to collect and how to do it, we also need to be mindful of potential privacy concerns (there's a reason browsers increasingly block indiscriminate tracking scripts!) and negative impacts on page load performance (3rd party JS scripts are notorious for slowing down page load times).

Relates to #483

**Comment by JohannaJoergensen** [comment]
_edited: typo_

**Comment by JohannaJoergensen** [comment]
2 Nov - James, Svetlana and Zoe to regroup to discuss initial recommendations

**Comment by james-nash** [comment]
24 Nov: Waiting on James & co to pull his finger out

**Comment by zdxd** [comment]
@james-nash @JohannaJoergensen is there a reason for moving this to backlog? It is in progress, but makes sense to not include in stand-ups around Dec 14th release as metrics will add to dev work and are not yet defined.

**Comment by JohannaJoergensen** [comment]
@zdxd @james-nash We can label this issue as future site work if not part of the soft launch

**Comment by zdxd** [comment]
Updated to website- future milestone

**Comment by JohannaJoergensen** [comment]
18 Jan - work in progress; work completed - feedback to follow

**Comment by zdxd** [comment]
same as above plus discussed with CX and recommend site team meet with them to discuss above
